tests/ctst/common/common.ts

```diff
@@ -15,3 +17,31 @@ Given('a {string} bucket', async function (versioning) {
         await S3.putBucketVersioning(this.getCommandParameters());
     }
 });
+
+Given('an existing bucket {string} {string} versioning', async function (bucketName, hasVersioning) {
```

Comment:
Do we want a bit more options in this base scenario? Like the following parameters:

* Object lock enabled
* If yes, if we enable or not the retention
* If retention is enabled, whether we want it in governance or compliance mode.

This way, we can better reuse this common step definition. Another alternative would be to create other steps for these scenarios, but it duplicates the code.

Reply:
22d2f30b793b72bbd174ea27a9693b1e6900d79e

this look ok?